client/app/components/sidebar-menu/sidebar-menu.less

```diff
@@ -10,8 +10,15 @@ button, toggle-class {
 }
 
 #sidebar-menu {
+    @sidebar-width: 300px;
+
+    width: @sidebar-width;
     z-index: 1020;
```

Comment:
Still useful ?

Reply:
About what `@sidebar-width` ?
It's used a bit further in the file.